CONTRIBUTING.md

```diff
@@ -58,3 +58,18 @@ Additionally, we have an org-wide base config for Renovate.
 [conventional-commits]: [https://www.conventionalcommits.org/]
 [semantic-versioning]: [https://semver.org/]
 [release-please]: [https://github.com/googleapis/release-please]
+
+### "Contrib" repositories
+
+Along with SDK repositories, we also maintain "contrib" repositories for most of our supported languages.
+These repositories are "monorepos" that house community contributions which are extensions to, or integrations with, the base SDKs.
+Examples include providers, hooks and framework-specific SDKs.
+In addition to the normal [repository requirements](#repository-requirements) (many of which are already implemented for all packages in each monorepo), components must each have _at least_ one CODEOWNER. It's the responsibility of the CODEOWNER(s) to:
+
+- review and resolve issues pertaining to their component
+- review and resolve pull requests pertaining to their component, including automated pull requests from dependabot, etc
+- release new versions of their component
+  - it's not required to implement new features in new releases, but releases containing dependency updates and bug fixes are expected
```

Comment:
```suggestion
 - it's not required to implement new features in new releases, but releases containing dependency updates and bug fixes are expected
```

Reply:
This was meant as a subpoint. At least on my editor locally that requires 2 spaces.